Update meeting-policies-in-teams.md (#4680)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/meeting-policies-in-teams.md

|Per-organizer and per-user     |When you implement a combination of a per-organizer and per-user policy, certain features are restricted for meeting participants based on their policy and the organizer's policy. For example, **Allow cloud recording** is a per-organizer and per-user policy. Turn on this setting to allow the meeting organizer and participants to start and stop a recording.

By default, a policy named Global (Org-wide default) is created. All users in your organization are assigned the Global meeting policy by default. You can either make changes to it or create one or more custom policies and assign users to them. Users will get the Global policy unless you create and assign a custom policy. When you create a custom policy, you can allow or prevent certain features from being available to your users, and then assign it to one or more users who will have the settings applied to them.

> -Note-

Reviewer: > [!NOTE]

> Meeting details buttom will be aviliable if a user has the audioconference licenses enabled or the user is allow for audioconferencing, if not, the meeting details will not be aviliable

Reviewer: ```suggestion
> The **Meeting Details** button will be available if the user has an Audio Conferencing license enabled or if the user is allowed for Audio Conferencing. If not, the meeting details will not be available.
```
